.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+on: [push]
+name: CI
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.19.x]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Modules tidy
+      run: |
+        go mod tidy
+        if [[ -n $(git status --porcelain) ]]; then echo "Please fix the issues with go mod tidy"; exit 1; fi
+    - name: Lint
+      run: |
+        go vet ./...
```

Comment:
should we use `mattermost-govet`?

Reply:
That has several MM related conventions. My next step is to incorporate golangci itself. This is just a basic skeleton.